inputFiles/poromechanics/impermeableFault_benchmark.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Included>
+     <File name="faultPoroelastic_base.xml"/>
+  </Included>
+
+<!-- SPHINX_POROMECHANICSSOLVER -->
+  <Solvers gravityVector="{0.0, 0.0, 0.0}">
+    <SinglePhasePoromechanics 
+      name="PoromechanicsSolver" 
+      solidSolverName="mechanicsSolver"
+      fluidSolverName="SinglePhaseFlowSolver"
```

Comment:
would be good to use a consistent capitilization style, e.g. `singlePhaseFlowSolver` and `poromechanicsSolver` to match `mechanicsSolver`

Reply:
Good suggestion